src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -24,37 +24,74 @@
+           from-network-values-sanitized (send-utils/network-values-sanitized from-network-values)
+           to-network-amounts            (send-utils/network-amounts {:route          chosen-route
+                                                                      :token-decimals token-decimals
+                                                                      :native-token?  native-token?
+                                                                      :to?            true})
+           to-network-values             (send-utils/network-values to-network-amounts)
+           to-network-values-sanitized   (send-utils/network-values-sanitized to-network-values)]
        {:db (-> db
                 (assoc-in [:wallet :ui :send :suggested-routes] suggested-routes-data)
                 (assoc-in [:wallet :ui :send :route] chosen-route)
+                (assoc-in [:wallet :ui :send :from-values-by-chain] from-network-values-sanitized)
+                (assoc-in [:wallet :ui :send :to-values-by-chain] to-network-values-sanitized)
                 (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))}))))
 
 (rf/reg-event-fx :wallet/suggested-routes-error
  (fn [{:keys [db]} [_error]]
    {:db (-> db
-            (update-in [:wallet :ui :send] dissoc :suggested-routes)
-            (update-in [:wallet :ui :send] dissoc :route)
+            (update-in [:wallet :ui :send] dissoc :suggested-routes :route)
             (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))}))
 
 (rf/reg-event-fx :wallet/clean-suggested-routes
  (fn [{:keys [db]}]
-   {:db (-> db
-            (update-in [:wallet :ui :send] dissoc :suggested-routes)
-            (update-in [:wallet :ui :send] dissoc :route)
-            (update-in [:wallet :ui :send] dissoc :loading-suggested-routes?))}))
+   {:db (update-in db
+                   [:wallet :ui :send]
+                   dissoc
+                   :suggested-routes
+                   :route
+                   :from-values-by-chain
+                   :to-values-by-chain
+                   :loading-suggested-routes?
+                   :suggested-routes-call-timestamp)}))
 
 (rf/reg-event-fx :wallet/clean-send-address
  (fn [{:keys [db]}]
    {:db (update-in db [:wallet :ui :send] dissoc :recipient :to-address)}))
 
+(rf/reg-event-fx :wallet/clean-disabled-from-networks
+ (fn [{:keys [db]}]
+   {:db (update-in db [:wallet :ui :send] dissoc :disabled-from-chain-ids)}))
+
 (rf/reg-event-fx
  :wallet/select-send-address
  (fn [{:keys [db]} [{:keys [address recipient stack-id start-flow?]}]]
    (let [[prefix to-address] (utils/split-prefix-and-address address)
          test-net?           (get-in db [:profile/profile :test-networks-enabled?])
          goerli-enabled?     (get-in db [:profile/profile :is-goerli-enabled?])
+         prefix              (if (string/blank? prefix) constants/default-address-prefix prefix)
```

Comment:
hmm, prefix is a bit generic imo. Can we make this more descriptive. multi-chain-prefix maybe? 🤔 

Also I wonder can we abstract the below code into some utility? 

e,g resolve-network-preferences? 🤔 
this
```
prefix              (if (string/blank? prefix) constants/default-address-prefix prefix)
prefix-seq          (string/split prefix #":")
selected-networks   (->> prefix-seq
                                  (remove string/blank?)
                                  (mapv #(utils/short-name->id (keyword %) test-net? goerli-enabled?)))
 ```